include/z64item.h

```diff
@@ -144,21 +144,21 @@ typedef enum {
 } InventorySlot;
 
 typedef enum ItemId {
-    /* 0x00 */ ITEM_OCARINA,
+    /* 0x00 */ ITEM_OCARINA_OF_TIME,
     /* 0x01 */ ITEM_BOW,
     /* 0x02 */ ITEM_ARROW_FIRE,
     /* 0x03 */ ITEM_ARROW_ICE,
     /* 0x04 */ ITEM_ARROW_LIGHT,
     /* 0x05 */ ITEM_OCARINA_FAIRY,
     /* 0x06 */ ITEM_BOMB,
     /* 0x07 */ ITEM_BOMBCHU,
-    /* 0x08 */ ITEM_STICK,
-    /* 0x09 */ ITEM_NUT,
+    /* 0x08 */ ITEM_DEKU_STICK,
+    /* 0x09 */ ITEM_DEKU_NUT,
     /* 0x0A */ ITEM_MAGIC_BEANS,
     /* 0x0B */ ITEM_SLINGSHOT,
     /* 0x0C */ ITEM_POWDER_KEG,
-    /* 0x0D */ ITEM_PICTO_BOX,
-    /* 0x0E */ ITEM_LENS,
+    /* 0x0D */ ITEM_PICTOGRAPH_BOX,
```

Comment:
Worth noting we decided on `PictoBox` before as the item name, but the item name texture itself is `Pictograph Box`, so I feel this might generate discussion.

Reply:
Oh no, this discussion again.
Personally I don't have a strong opinion in either way. Matching the in-game item name / texture sounds fine to me.